app/assets/stylesheets/alchemy/elements.scss

```diff
@@ -384,11 +387,11 @@
 
       .icon { margin-left: 1px }
     }
+  }
 
-    &.disabled {
-      @include opacity(0.3);
-      cursor: default;
-    }
+  a.disabled, > span.icon {
```

Comment:
Avoid qualifying class selectors with an element.<br>Each selector in a comma sequence should be on its own line